### ISSUE TITLE
Compare files

### ISSUE THREAD

**Issue by leibhold** [leibhold/Signs#23]
![image](https://user-images.githubusercontent.com/6262140/87840435-d29cae80-c8de-11ea-9526-06b2d5176c92.png)

**Comment by galen-yaham** [comment]
Yes this function campare the upload file and original file if there is difference. then it will alarm and gives indication to upload again. 
But this function is not so impotant, it can be deleted from the software. 
(I'm Galen from Yaahm and try to study how to use github)
